### DISCUSSION TITLE
ILI9488 GPIO output gpio_num error

### DISCUSSION THREAD

**Discussion by LukasLuck** [Bodmer/TFT_eSPI#2188] [Q&A - Touch]
Hello Bodmer,
I am using ILI9488 and ESP32. I have the PIN connections as below and I have modified user_setup.h accordingly.

I tried to run Test_Touch_Controller.ino and get responses on serial monitor such as:
Starting...
E (10) gpio: gpio_set_level(226): GPIO output gpio_num error
E (10) gpio: gpio_set_level(226): GPIO output gpio_num error
E (21) gpio: GPIO can only be used as input mode
E (73) gpio: gpio_set_level(226): GPIO output gpio_num error
E (457) gpio: gpio_set_level(226): GPIO output gpio_num error
E (457) gpio: gpio_set_level(226): GPIO output gpio_num error
x: 0     y: 0     E (579) gpio: gpio_set_level(226): GPIO output gpio_num error
E (579) gpio: gpio_set_level(226): GPIO output gpio_num error

LED on ESP32 is blinking continuously.

Pressing on display, but nothing changes on serial monitor

![Bild](https://user-images.githubusercontent.com/98472485/202799122-c062fbe7-c67f-44bd-82b9-8f6135a48ce3.png)

**My question:**
1) what would be the cause of that error (how to fix)?
2) Do we need to define T_IRQ, LED pins in script/ user_setup

Thank you very much in advance.

Best regards

Lukas

**Comment by Bodmer** [comment]
Typically this error means the GPIO number was out of range or specified as -1

T_IRQ is  not used bt the library
If the LED control pin is connected to the processor then the pin should be specified in the setup.LED
